pkg/datapath/cells.go

```diff
@@ -144,4 +153,9 @@ type datapathParams struct {
 	BpfMaps []bpf.BpfMap `group:"bpf-maps"`
 
 	NodeMap nodemap.Map
+
+	// Depend on DeviceManager to ensure devices have been resolved.
+	// This is required until option.Config.GetDevices() has been removed and
+	// uses of it converted to Table[Device].
+	DeviceManager *linuxdatapath.DeviceManager
```

Comment:
This really annoys me. As long as we have globals like this we need to play dangerous games with the initialization ordering to get them initialized early enough. Next step is definitely the prompt removal of `option.Config.GetDevices` to make sure no new module accidentally accesses it uninitialized.